stream-chat-android-client/api/stream-chat-android-client.api

```diff
@@ -2694,23 +2893,6 @@ public final class io/getstream/chat/android/client/uploader/ProgressTrackerKt {
 	public static final fun toProgressCallback (Lio/getstream/chat/android/client/uploader/ProgressTracker;)Lio/getstream/chat/android/client/utils/ProgressCallback;
 }
 
-public final class io/getstream/chat/android/client/utils/FilterObject {
```

Comment:
There are gonna be a lot of Java clients out there who reference this old package/class, whenever they wrote down

```java
FilterObject filter = Filters.in(...);
```

Is there no way for us to avoid this huge breaking change for them? This is almost all clients who use Java + create a filter at any point.

Reply:
Old package doesn't make sense and it is something we need to do at some point.
I think change it now make sense, They only need to update the import line as it is explained on [here](https://github.com/GetStream/stream-chat-android/wiki/Migration-guide:-FilterObject)

Reply:
I'd prefer a softer breaking change if it's possible to do, maybe leaving a deprecated interface behind in the old package that the new things implement for compatibility... But if you want to go with the hard breaking change, that's fine too.